File: build_sample/python/humancat/human.py

class Human(object):
    def __init__(self, name):
        self.name = name
        self.age = 20

    def get_name(self):
        return self.name
    
    def set_name(self, name):
        self.name = name

    def get_age(self):
        return self.age
    
    def set_age(self, age):
        self.age = age

    def get_info(self):
        return f'My name is {self.name} and {self.age} years old.'

File: build_sample/js/humancat/lib/human.js

class Human {

    constructor(name) {
        this.name = name;
        this.age = 20;
    }

    GetName() {
        return this.name;
    }

    SetName(name) {
        this.name = name;
    }

    GetAge() {
        return this.age;
    }

    SetAge(age) {
        this.age = age;
    }
    
    GetInfo() {
        return `My name is ${this.name} and ${this.age} years old.`;
    }
}

module.exports = Human;
